src/alchemlyb/visualisation/convergence.py

```diff
@@ -0,0 +1,72 @@
+import matplotlib.pyplot as plt
+from matplotlib.font_manager import FontProperties as FP
+import numpy as np
+
+def plot_convergence(forward, forward_error, backward, backward_error,
+                     units='kBT', ax=None):
+    """Plot the forward and backward convergence.
+
+    Parameters
+    ----------
+    forward : List
+        A list of free energy estimate from the first X% of data.
+    forward_error : List
+        A list of error from the first X% of data.
+    backward : List
+        A list of free energy estimate from the last X% of data.
+    backward_error : List
+        A list of error from the last X% of data.
+    units : str
+        The label for the unit of the estimate. Default: :math:`kBT`
```

Comment:
Do not use math mode for kBT. This is a Python string so the default should be what you can provide to the function, i.e., "kBT".

Reply:
An can we use "kT" instead of "kBT"? (See comment elsewhere)

Reply:
Add a note that setting the unit does not convert anything – this is purely for setting labels. The data already need to be in the desired unit.

Reply:
@orbeckst Thank you for the suggestion. In https://github.com/alchemistry/alchemlyb/pull/114#discussion_r603577969 you mentioned that kT should be formatted as latex. I initially thought that it means using the math model but I guess this is not the case. I wonder if I can have an example of how this should be formatted? Thank you.

Reply:
I've been confused about what the argument does. Given that it's just a label, it should be clear that this is a Python string. You could set it to `r"$k_B R$"`, which does LaTeX formatting in matplotlib. It should always be clear that the argument is a string. 

Sorry if this was confusing.